README.md

```diff
@@ -31,6 +32,15 @@ Have something to add or change? Open a [pull request](https://github.com/luqmaa
   - [GTFS Realtime Utilities](#gtfs-realtime-utilities)
 - [SIRI](#siri)
 
+### Getting started
+
+If this is your first time dealing with transit data, you might find these links useful:
+
+- [GTFS](https://developers.google.com/transit/gtfs/) - GTFS is a of text files that contains infrequently changing transit data, like stops, routes, trips, and other schedule data. Most transit agencies typically update their GTFS feed every few months.
+- [GTFS Realtime](https://developers.google.com/transit/gtfs-realtime/) - GTFS Realtime consists of three binary files that contain realtime vehicle positions, realtime arrival information, and service alerts. Transit agencies typically update these files every minute.
+- [Transitland ](https://transit.land/) - A community curated list of GTFS/GTFS-realtime data feeds from around the world. If you're trying to get realtime data for some agency, this is a good place to start.
```

Comment:
I don't think Transitland had any realtime feeds yet, although it's on their roadmap. Transitfeeds.com is the best directory for realtime feeds that I know of.